Formula/s/sitefetch.rb

```diff
@@ -0,0 +1,21 @@
+class Sitefetch < Formula
+  desc "Fetch an entire site and save it as a text file"
+  homepage "https://github.com/egoist/sitefetch"
+  url "https://registry.npmjs.org/sitefetch/-/sitefetch-0.0.17.tgz"
+  sha256 "eb65e7d0179e5c06f70a3071865cf8199827428adb64da19de12afaf682c0ddd"
+  license "MIT"
+
```

Comment:
want to add a `head` url?
```suggestion
  head "https://github.com/egoist/sitefetch.git", branch: "main"
```

Reply:
the head build does not work.